packages/fxa-auth-server/lib/oauthdb/index.js

```diff
@@ -99,7 +101,19 @@ module.exports = (log, config) => {
 
     async grantTokensFromAuthorizationCode(oauthParams) {
       try {
-        return await api.grantTokensFromAuthorizationCode(oauthParams);
+        const codeResponse = await api.grantTokensFromAuthorizationCode(
+          oauthParams
+        );
+        const scopeSet = ScopeSet.fromString(codeResponse.scope);
```

Comment:
nit: I think this deserves a bit more of a comment about what it's doing and why, for context for future readers. Something about how sessionTokens live in the auth-server db so we have to create them here after the oauth-server has validated the request.